website/demo/index.html

```diff
@@ -82,6 +99,24 @@
         margin-top: 4px; /* height of SVG */
         position: absolute;
       }
+      @media (prefers-color-scheme: dark) {
+        a:link {
+          color: #3398e0;
+        }
+        a:visited {
+          color: #979ef8;
+        }
+        #code-input {
+          color: white;
+        }
+        body {
+          background-color: #002b36;
+          color: white;
+        }
```

Comment:
Isn't this already in main.css? Why duplicate the rules?

Reply:
Duplicated rules have been deleted.